blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js

```diff
@@ -58,10 +58,12 @@ export default function SimulationScreen ({ open, close }) {
   const stitle = useSelector(state => state.netlistReducer.title)
   const taskId = useSelector(state => state.simulationReducer.taskId)
   const [isResult, setIsResult] = useState(false)
+  const [logs, setlog] = useState(false)
   const graphsRef = useRef([])
   const datapointsRef = useRef([])
   const timeoutRef = useRef(null)
   const [noOfGraphs, setNoOfGraphs] = useState(0)
+  const [error, seterror] = useState(0)
```

Comment:
```suggestion
  const [error, setError] = useState('')
```
use camelCasing

set to empty string